src/models/db/adjustments.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+const _ = require('lodash')
+const assert = require('assert')
+const TABLE_NAME = 'L_TRANSFER_ADJUSTMENTS'
+const getAccountId = require('./accounts').getAccountId
+const getAccountById = require('./accounts').getAccountById
+const db = require('./utils')()
+
+function convertFromPersistentAdjustment (data, options) {
+  return getAccountById(data.ACCOUNT_ID, options).then((account) => {
+    return _.omit({
+      account: account.name,
+      amount: Number(data.AMOUNT).toString(),
+      authorized: Boolean(data.IS_AUTHORIZED),
+      memo: data.MEMO
+    }, (x) => !x)
+  })
+}
+
+function convertFromPersistent (rows, options) {
+  const debits = Promise.all(rows.filter((row) => row.DEBIT_CREDIT === 'debit')
+    .map((debit) => convertFromPersistentAdjustment(debit, options)))
+  const credits = Promise.all(rows.filter((row) => row.DEBIT_CREDIT === 'credit')
+    .map((credit) => convertFromPersistentAdjustment(credit, options)))
+  return Promise.all([debits, credits]).then((results) => {
+    if (results[0].length === 0 && results[1].length === 0) {
+      return {}
+    }
+    return {
+      debits: _.sortBy(results[0], (adjustment) => adjustment.account),
+      credits: _.sortBy(results[1], (adjustment) => adjustment.account)
+    }
+  })
+}
+
+function convertToPersistentAdjustment (transferId, type, data, options) {
+  return getAccountId(data.account, options).then((accountId) => {
+    return {
+      TRANSFER_ID: transferId,
+      ACCOUNT_ID: accountId,
+      DEBIT_CREDIT: type,
+      AMOUNT: data.amount,
+      IS_AUTHORIZED: Number(data.authorized || false),
```

Comment:
Do we update rows in this table? If so, we need to explicitly pass `IS_AUTHORIZED` on all updates, or it will be set to `false`.